src/main/scala/nl/knaw/dans/easy/fedora2vault/EasyFedora2vaultApp.scala

```diff
@@ -41,18 +43,26 @@ class EasyFedora2vaultApp(configuration: Configuration) extends DebugEnhancedLog
   private lazy val ldap = new Ldap(ldapContext)
   private val emdUnmarshaller = new EmdUnmarshaller(classOf[EasyMetadataImpl])
 
-  def simpleTransForms(input: File, outputDir: File): Try[FeedBackMessage] = {
-    input.lineIterator.filterNot(_.startsWith("#")).map(datasetId => {
-      simpleTransform(datasetId, outputDir)
-        .doIfFailure { case t =>
-          logger.error(s"$datasetId failed", t)
+  def simpleTransForms(input: File, outputDir: File)(implicit printer: CSVPrinter): Try[FeedBackMessage] = {
+    input.lineIterator.filterNot(_.startsWith("#")).map { datasetId =>
+      val uuid = UUID.randomUUID
+      simpleTransform(outputDir / uuid.toString)(datasetId)
+        .doIfFailure { case t => logger.error(s"$datasetId -> $uuid failed: $t", t) }
+        .recoverWith {
+          case t: FedoraClientException if t.getStatus != 404 => Failure(t)
+          case t => CsvRecord(
+            datasetId, doi = "", depositor = "", SIMPLE, uuid, s"FAILED: $t"
+          ).print
         }
-    }).collectFirst { case t @ Failure(_) => t }
-      .getOrElse(Success(s"All datasets in ${ input } saved as bags in ${ outputDir }"))
-    // TODO return individual feedback messages preceded with FeedBack.header
-  }
+    }
+  }.collectFirst { case f @ Failure(_) => f }
+    .getOrElse(Success(
+      s"""All datasets in $input
+         | saved as bags in $outputDir
+         | """.stripMargin
```

Comment:
Is the newline at the end necessary?

```suggestion
      s"""All datasets in $input
         | saved as bags in $outputDir""".stripMargin
```